.github/ISSUE_TEMPLATE/NEW_NOTEBOOK.yml

```diff
@@ -27,10 +26,10 @@ body:
     attributes:
       label: Module/Software/IOOS data service?
       description: Can you provide a minimum example of the expected code and results in a notebook?
-      placeholder: ""
+      placeholder: Yes, see this gist.
   - type: input
     id: description
     attributes:
       label: Provide a description.
       description: Please provide a detailed description of the suggested example below.
-      placeholder: ""
+      placeholder: The notebook should detail how to bring source data in, and create an interactive graphical timeseries of the data.
```

Comment:
Should we add in the placeholder: `Example: phrase goes here` to avoid users thinking that it is limited to the scope of the phrase. For example, reading it here gives the impression that it must be an interactive graphical representation of the data?

Reply:
good point. we could remove `placeholder`...

Reply:
I like it! I just want to make sure folks understand that they are a text example, not a guide.